packages/relay/tests/lib/sdkClient.spec.ts

```diff
@@ -271,4 +272,1815 @@ describe('SdkClient', async function () {
       }
     });
   });
+
+  describe('Rate Limiter', async () => {
+    const transactionBuffer = new Uint8Array([
```

Comment:
nit: better approach would have been to just read in a file with some details. Actually contract byte code from one of the test  contracts would have been appropriate here

Reply:
Resolving but @ebadiere we should resolve this in follow up